src/Raven.Client/Documents/Session/SessionOptions.cs

```diff
@@ -8,6 +8,12 @@ public enum TransactionMode
         ClusterWide,
     }
 
+    public enum ShardedBatchBehavior
+    {
+        SingleBucket,
+        MultiBucket
```

Comment:
This means allow multi bucket operation. (Need better naming here).
That should be the default, I think. Given that old clients talking to 6.0 will not pass it.